test/e2e/framework/pv_util.go

```diff
@@ -48,6 +48,12 @@ const (
 	VolumeSelectorKey = "e2e-pv-pool"
 )
 
+var (
+	// Common selinux labels
+	SelinuxLabel = &v1.SELinuxOptions{
```

Comment:
sorry, just noticed this, it should be `SELinuxLabel`.

/lgtm cancel

(and you might consider moving it into storage/utils)